2-mandatory/3-financial-times.js

```diff
@@ -1,20 +1,38 @@
 /*
-    Imagine you are working on the Financial Times web site! They have a list of article titles stored in an array.
+    Imagine you are working on the Financial Times web site! 
+    They have a list of article titles stored in an array.
 
-    The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
-    Implement the function below, which will return a new array containing only article titles which will fit.
+    The home page of the web site has a headline section, 
+    which only has space for article titles which 
+    are 65 characters or less.
+    Implement the function below, which will return a new array containing only article titles 
+    which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
     // TODO
+    let articleTitles = [];
+    for (articles of allArticleTitles) {
+        if (articles.length <= 65){
+            articleTitles.push(articles);
+        }
+    }
+    return articleTitles;
 }
-
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
     // TODO
+let stringWithFewestWords=''
+for (let i = 0; i < allArticleTitles.length; i++) {
+if (stringWithFewestWords.length < 1 ||
+    stringWithFewestWords.split(" ").length > allArticleTitles[i].split(" ").length) {
+        stringWithFewestWords = allArticleTitles[i];
+}
+}
+return stringWithFewestWords;
 }
```

Comment:
You could try initialising stringWithFewestWords with the first title, then you would only need to check that each subsequent word is shorter.